src/api/db/structure.sql

```diff
@@ -952,10 +952,12 @@ CREATE TABLE `projects` (
   `kind` enum('standard','maintenance','maintenance_incident','maintenance_release') CHARACTER SET utf8 COLLATE utf8_bin DEFAULT 'standard',
   `url` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL,
   `required_checks` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `staging_workflow_id` int(11) DEFAULT NULL,
```

Comment:
good thing you discarded adrian's review because it's just WIP in a different branch

Reply:
We want to deploy the current version to not block deployments for a week. I know it's not optimal, but this is what our current setup allows us.

We will iterate and realign if needed next Thursday (so the next maintenance window).

Reply:
this design is flawed and breaks basic concepts. 

You waste effort in basing on this if these basic issues are not figured out first.

This makes the current master state basically not releasable.

So revert and discuss the concept with relevant stake holders first. 

This is not a whish.

Reply:
I can't say I see a rush to deploy this. This is not a bug fix, this is a concept - in its very early stage. So to me this sounds like treating production as playground.

Reply:
> this design is flawed and breaks basic concepts.
You waste effort in basing on this if these basic issues are not figured out first.

Can please elaborate a bit? In which cases adding a reference is breaking basic concepts and doing everything a mess? I don't really get it. You need to explain this into deep.

> This makes the current master state basically not releasable.

Why?

> So revert and discuss the concept with relevant stake holders first.

For reverting things Github provides a beautiful button you can also click if you think is needed, after all it creates another PR that we need to review.

We wanted to discuss this with you as I wrote in my dismiss of your review in the other PR. But unluckily we didn't found you in your table 4 times in a day... We need to talk that's for sure because you are complaining without explaining too much.

> This is not a whish.

Yes mi lord! Is this meaning that is a command? :thinking: 

> I can't say I see a rush to deploy this. This is not a bug fix, this is a concept - in its very early stage.

Sure we shouldn't merge this with master, is our fault. :+1: 

> treating production as playground

You mean by monkey patching or by pushing to master like others did before? :dagger: